src/Explorer/Tabs/MongoShellTab/MongoShellTabComponent.tsx

```diff
@@ -135,7 +136,7 @@ export default class MongoShellTabComponent extends Component<
     const databaseId = this.props.collection.databaseId;
     const collectionId = this.props.collection.id();
     const apiEndpoint = configContext.BACKEND_ENDPOINT;
-    const encryptedAuthToken: string = userContext.accessToken;
+    const encryptedAuthToken: EncryptedAccessToken = userContext.accessToken;
```

Comment:
Have you updated the mongo shell to accept an `EncryptedAccessToken` type? If not, then we should We should pass `encryptedAuthToken.primaryToken` to the iframe instead.